src/main/resources/config/config.yaml

```diff
@@ -27,10 +27,20 @@ jerseyClientConfig:
   disabledSecureConnection: ${DISABLE_INTERNAL_HTTPS}
 
 rateLimiter:
-  rate: ${RATE_LIMITER_VALUE:-25}  # the rate is for all requests except POST
-  rateForPost: ${RATE_LIMITER_VALUE_POST:-5}
+  rate: ${RATE_LIMITER_VALUE:-75}  # the rate is for all requests except POST. Rate is across all publicapi instances.
```

Comment:
this `rate` looks like it should be `numberOfRequests`? the actual rate is calculated by `numberOfRequests` divided by `perMillis`.